### PR TITLE
Add cleaner mod

### PR DESCRIPTION
**Pull request by kilbith**
This will remove all unknown nodes / items inside a radius of 16.

### CODE REVIEW

File: games/default/files/xpanes/depends.txt [hunk 1]
```diff
@@ -1 +0,0 @@
-default
```
Comment (kilbith) [inline games/default/files/xpanes/depends.txt]: This is necessary to keep this file otherwise I cannot start the game.

Comment (MoNTE48) [reply games/default/files/xpanes/depends.txt]: Damn, now I understand my problem! I thought these files were no longer used and were replaced by mod.conf

**Comment by MoNTE48** [comment]
Unfortunately, unknown items were not removed from my inventory...

**Comment by kilbith** [comment]
> Unfortunately, unknown items were not removed from my inventory...

Replace `stack:clear()` by : `inv:set_stack("main", i, "")`. This will fix the issue.

**Comment by MoNTE48** [comment]
Unfortunately, it did not help again. I disabled the mesecons mod on the server, and there are still unknown items in the inventory

**Comment by MoNTE48** [comment]
```local entname = obj:get_entity_name()```
Deprecated call to "get_entity_name 
Need change it on get_luaentity(), yes?

**Comment by kilbith** [comment]
> Need change it on get_luaentity(), yes?

No, this won't work.

**Comment by MoNTE48** [comment]
Working for me... 




пятница, 24 мая 2019 г., 14:23 +0200 от notifications@github.com  <notifications@github.com>:

**Pull request merged**